crates/ruma-events/src/room/name.rs

```diff
@@ -64,12 +93,14 @@ mod tests {
     use serde_json::{from_value as from_json_value, json, to_value as to_json_value};
 
     use super::NameEventContent;
-    use crate::{StateEvent, Unsigned};
+    use crate::{room::name::RoomName, StateEvent, Unsigned};
 
     #[test]
     fn serialization_with_optional_fields_as_none() {
         let name_event = StateEvent {
-            content: NameEventContent { name: Some("The room name".into()) },
+            content: NameEventContent {
+                name: RoomName::try_from("The room name".to_string()).ok(),
```

Comment:
This is a personal pet peeve of mine.. See also clippy lint [str_to_string](https://rust-lang.github.io/rust-clippy/master/index.html#str_to_string).

```suggestion
                name: RoomName::try_from("The room name".to_owned()).ok(),
```